src/components/ErrorBoundary/index.tsx

```diff
@@ -1,66 +1,157 @@
 import { Trans } from '@lingui/macro'
 import * as Sentry from '@sentry/react'
-import React, { PropsWithChildren } from 'react'
+import { ButtonLight, ButtonPrimary } from 'components/Button'
+import { ChevronUpIcon } from 'nft/components/icons'
+import { useIsMobile } from 'nft/hooks'
+import React, { PropsWithChildren, useState } from 'react'
+import { Copy } from 'react-feather'
 import styled from 'styled-components/macro'
 
-import { ExternalLink, ThemedText } from '../../theme'
-import { AutoColumn } from '../Column'
-import { AutoRow } from '../Row'
+import { CopyToClipboard, ExternalLink, ThemedText } from '../../theme'
+import { Column } from '../Column'
 
 const FallbackWrapper = styled.div`
   display: flex;
-  flex-direction: column;
-  width: 100%;
-  align-items: center;
-  z-index: 1;
+  width: 100vw;
+  height: 100vh;
 `
 
 const BodyWrapper = styled.div<{ margin?: string }>`
+  width: min(100%, 500px);
+  margin: auto;
   padding: 1rem;
+`
+
+const SmallButtonPrimary = styled(ButtonPrimary)`
+  width: auto;
+  font-size: 16px;
+  padding: 10px 16px;
+  border-radius: 12px;
+`
+
+const SmallButtonLight = styled(ButtonLight)`
+  font-size: 16px;
+  padding: 10px 16px;
+  border-radius: 12px;
+`
+
+const StretchedRow = styled.div`
+  display: flex;
+  gap: 24px;
+
+  > * {
```

Comment:
i am not a fan of child selectors in styled components. better to create a shared style and extend the components for the children imo

Reply:
Not a fan either, but I somewhat disliked the alternative approach of wrapping all children in a StyledComponent to give it `flex: 1, display: flex`. Don't really have a strong opinion here